Examples/Algorithms/Geant4/src/Geant4Simulation.cpp

```diff
@@ -84,6 +88,15 @@ ActsExamples::Geant4Simulation::Geant4Simulation(
   G4EventManager::GetEventManager()->GetStackManager()->SetVerboseLevel(
       geantVerboseLevel);
 
+  // Suppress the printing of physics information.
+  if (m_cfg.printPhysicsParameters) {
+    G4HadronicParameters::Instance()->SetVerboseLevel(0);
```

Comment:
now clang tidy complains that this does not exist :disappointed: 

maybe we can just move everything into the version switch?

Reply:
yeah lets do this